prog2.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python
+
+import paramiko
+import threading
+import time
+
+com = raw_input("Please specify the command to run: ")
```

Comment:
I would suggest to used `argparse` module instead to parse command line parameters